### DISCUSSION TITLE
Having issues getting the voice acting to work

### DISCUSSION THREAD

**Discussion by sylvanheart89** [Albeoris/Memoria#512] [Q&A] [answered]
I just got Final Fantasy 9 and added the Memoria patch from snouz that came out a few weeks back. I've tried older versions too, but I can't get any of the voice-overs to play in the game. I'm not sure where to start or who to ask for help since I have no idea what's causing the issue. It might be a setting I haven't enabled or something on my computer that's not set up right. Is the voice acting just supposed to work after I patch Memoria? The background music works fine, but I'm not hearing much more than that. I guess I haven't given much to go on here but if anyone has any ideas.. I'm out of options.

**Comment by snouz** [comment, marked as answer]
Hi, Memoria adds voice-over _**capababilities**_ to modders, for an example of actual voice-acting, you can google Echo-S, which is a project that utilizes this functionality.

**Reply by sylvanheart89** [reply]
Ohh!

sorry I'm just dumb then I thought the memoria tool was what adds the voice-over into the game.

**Reply by sylvanheart89** [reply]
I looked into Echo-S and was also playing around with the whole voiceover system to try and learn some of the basics. I couldn't really get too far. It says the game identifies a line like "Cards!? Right now?" as VA_373.ogg. When I start up the game I can see the log file showing something like this:

19.02.2025 09:47:10 |M| [VoiceActing] field:2, msg:51, text:Baku
19.02.2025 09:47:10 |M| “Here’s the plan!”, path(s):'Voices/US/2/va_51', 'Voices/US/2/va_51_Baku', 'Voices/US/2/va_51_3' (not found)

So I guess it wants me to put an .ogg file called va_51 in "Voices/US/2". Is that it? If I'm just supposed to boot up the game and hear the dialog from that ogg file then mission accomplished, but nothing's playing. Is there more to it?

**Reply by Tirlititi** [reply]
Yes. More precisely, the complete path must be (from the game's directory):
```txt
{VoiceMod}/StreamingAssets/Assets/Resources/Sounds/Voices/US/2/va_51.ogg
```

**Reply by Tirlititi** [reply]
``{VoiceMod}`` must be replaced by the name of your voice mod folder (see [here](https://github.com/Albeoris/Memoria/wiki/Mod-folders)).
You need to activate it in the Mod Manager. You also need to activate the section ``[VoiceActing]`` in ``Memoria.ini``, but you have done that already I suppose.

**Reply by sylvanheart89** [reply]
the link you sent says i need to change the folder names under the [Mod] section in memoria.ini, so i changed the folder in the Final Fantasy IX directory to "VoiceTest" and also changed it in memoria.ini

do the "MoguriFiles" and "MoguriSoundtrack" folders need to exist for this to work? maybe its not working because something is missing or corrupt in my game folder? the [VoiceActing] section is also set to 1 in the ini file.

**Reply by Tirlititi** [reply]
You can activate mods with the Mod Manager's interface instead of editing the ``Memoria.ini`` file manually. The link (tries to) explain how that works, so you could debug it yourself. But I think I shouldn't have given you that link in the end; it looks like it adds more confusion than anything else.

So use the Mod Manager instead (it's a tab in the launcher).
It may be that your problem is because you renamed your folder after the Mod Manager registering it and now its file ``VoiceTest/ModDescription.xml`` is not accurate anymore. Try deleting that file (that was generated automatically) and then open the Mod Manager.

Non-existing folders registered in the INI won't cause any bug, but will simply have no effect. It is strange that you have these registered in the INI if they don't exist though...

**Reply by sylvanheart89** [reply]
ill try removing final fantasy 9 completely and then delete any of the remaining files/folders and then reinstall it from scratch, maybe that'll work.

**Reply by sylvanheart89** [reply]
here's what i did:

i deleted Final Fantasy 9 from steam and then deleted all of the remaining files since it still showed "FINAL FANTASY IX" in the steamapps folder. i also checked AppData just to be safe, there's nothing there out of the ordinary. i reinstalled Memoria from here using 2024.12.04 Memoria.Patcher.exe:

https://github.com/Albeoris/Memoria/releases

i ran Memoria.Patcher.exe as an admin, it said it was successful and when i load it i can see the launcher. there is nothing in the "Mods" tab when i load it, so i close the launcher and then i create a folder structure like this:

FINAL FANTASY IX\VoiceTest\StreamingAssets\Assets\Resources\Sounds\Voices\US\2

when i go back to the launcher, i can now see "VoiceTest" in the "Mods" tab, and clicking on the black circle turns it into a blue circle. the "VoiceActing" section in the ini file was set to 0 on a new install, so i also changed that to 1. 

somehow after doing all that, it started working so maybe something was just previously broken on my end.

**Comment by sylvanheart89** [comment]
are the brackets around "VoiceMod" supposed to be there? I put in the directory like this:

FINAL FANTASY IX/VoiceMod/StreamingAssets/Assets/Resources/Sounds/Voices/US/2/va_51.ogg

for the va_51.ogg file I just recorded some random stuff into my mic and saved it as an .ogg, but if i'm supposed to hear it in the game when Baku says his line, its just quiet. I'm not hearing anything other than the game audio. is there something else i need to set up that i'm not seeing?
